src/main/java/org/broadinstitute/hellbender/metrics/MultiLevelMetrics.java

```diff
@@ -1,17 +1,31 @@
 package org.broadinstitute.hellbender.metrics;
 
-import htsjdk.samtools.metrics.MetricBase;
+import org.broadinstitute.hellbender.tools.dataflow.transforms.metrics.SerializableMetric;
 
-public abstract class MultiLevelMetrics extends MetricBase {
-     /** The sample to which these metrics apply.  If null, it means they apply
+import java.util.Comparator;
+
+public abstract class MultiLevelMetrics extends SerializableMetric{
+    private static final long serialVersionUID = 1l;
+
+    /** The sample to which these metrics apply.  If null, it means they apply
      * to all reads in the file. */
     public String SAMPLE;
 
     /** The library to which these metrics apply.  If null, it means that the
      * metrics were accumulated at the sample level. */
-    public String LIBRARY = null;
+    public String LIBRARY;
 
     /** The read group to which these metrics apply.  If null, it means that
      * the metrics were accumulated at the library or sample level.*/
-    public String READ_GROUP = null;
+    public String READ_GROUP;
+
+    /**
+     * Generates a comparator to sort by Sample, ReadGroup, and then Library.
+     * Intended to be used by subclasses to construct their own comparators.
+     */
+    public static <T extends MultiLevelMetrics> Comparator<T> getMultiLevelMetricsComparator(){
```

Comment:
This could use a little more commenting. It look me a minute to really unpack this thing (I was tripped up by not knowing that Comparator.comparing is taking a sort key extracting function).